app-misc/figlet/figlet-222.recipe

```diff
@@ -5,35 +5,48 @@ and it can kern and 'smush' the letters these letters together in \
 various ways."
 HOMEPAGE="http://www.figlet.org/"
 COPYRIGHT="1991, 1993, 1994 Glenn Chappell and Ian Chai
-	1996, 1997, 1998, 1999, 2000, 2001 John Cowan
-	2002 Christiaan Keet"
+    1996, 1997, 1998, 1999, 2000, 2001 John Cowan
+    2002 Christiaan Keet"
 LICENSE="AFL v2.1"
 REVISION="1"
-SOURCE_URI="ftp://ftp.figlet.org/pub/figlet/program/unix/figlet222.tar.gz"
-CHECKSUM_MD5="5f84ad52d092d5db4ad5916df767646b"
-STATUS_HAIKU="stable"
-DEPEND=""
+SOURCE_URI="http://ftp.nsysu.edu.tw/FreeBSD/ports/local-distfiles/obrien/figlet222.tar.gz"
+CHECKSUM_SHA256="21f8f9e9b2190fbc0f801fb4704355e5d73d36c55670272cd08cad3408d1f5fe"
+SOURCE_DIR="figlet222"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+    figlet = $portVersion
+    cmd:figlet =$portVersion
+    cmd:chkfont
+    cmd:figlist
+    cmd:showfigfonts
+    "
+REQUIRES="
+    haiku
+    "
+
+BUILD_REQUIRES="
+    haiku_devel
+    "
+BUILD_PREREQUIRES="
+    cmd:gcc
+    cmd:make
+    "
 
 BUILD()
 {
-	cd figlet222
-	make
+    MAN=$manDir/man6
+    FONTS=$dataDir/figlet/fonts
+    DEST=$binDir
+    mkdir -p  ${MAN} ${FONTS}
+    make MAN=${MAN} FONTS=${FONTS}
 }
 
 INSTALL()
 {
-	cd figlet222
-	DISTRODIR=${DESTDIR}
-	MANDIR=${DISTRODIR}/`finddir B_APPS_DIRECTORY`/figlet/share/man
-	DEFAULTFONTDIR=${DISTRODIR}/`finddir B_APPS_DIRECTORY`/figlet/fonts
-	DESTDIR=${DISTRODIR}/`finddir B_APPS_DIRECTORY`/figlet
-	mkdir -p ${DESTDIR}/tmp
-	mkdir -p ${MANDIR}/man6
-	mkdir -p ${DEFAULTFONTDIR}/tmp
-	make install DESTDIR=${DESTDIR} MANDIR=${MANDIR} \
-		DEFAULTFONTDIR=${DEFAULTFONTDIR}
-	chmod 755 ${DESTDIR}/showfigfonts
-	chmod 755 ${DESTDIR}/figlist
-	rm -rf ${DESTDIR}/tmp
-	rm -rf ${DEFAULTFONTDIR}/tmp
+    make install MAN=${MAN} FONTS=${FONTS}
+    chmod 755 $binDir/showfigfonts
+    chmod 755 $binDir/figlist
+    chmod 755 $binDir/chkfont
 }
```

Comment:
> no such file or directory for /chkfont

I don't know the current state of your recipe.
make sure to remove the "work" folder before re-building the package. Otherwise past failed builds may interfere.